libexec/probes/worker-scripts/uploader/rabbitmquploader.py

```diff
@@ -80,6 +87,9 @@ def publishRToMq(self, arguments, event_types, datapoints):
             msg_body['datapoints'] = datapoints[event]
             self.SendMessagetoMQ(msg_body)
             
+    def restartPikaConnection(self):
+        self.connection = pika.BlockingConnection(self.parameters)

```

Comment:
Add a block to disconnect any existing channel and connection:
```
if self.channel:
   try:
      self.channel.close()
   except:
      ... add log line
```
and then analogously for `self.connection`.